src/components/search/search.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SearchView } from './search';
+
+const noop = jest.fn();
+
+describe('Search', () => {
+  test('renders SearchSnapshot', () => {
+    const { asFragment } = render(<SearchView onSearch={noop} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should call on search with userText', () => {
+    const onSearch = jest.fn();
+    render(<SearchView onSearch={onSearch} />);
+    const input = screen.getByRole('textbox');
+    userEvent.type(input, 'test');
+    const button = screen.getByRole('button');
+    userEvent.click(button);
+
+    expect(onSearch).toHaveBeenCalledWith('test');
+  });
+
+  it('should set value to input if searchQuery', async () => {
```

Comment:
1. Is `async` modifier required here?
2. Not really clear description name for your test: for person not familiar with app `searchQuery` might be not clear. Whenever you specify some prop/argument/method add appropriate description. In you case it should be `should set value to input if "searchQuery" prop exists and not empty`